app/src/test/java/org/mozilla/fenix/home/HomeMenuBuilderTest.kt

```diff
@@ -0,0 +1,234 @@
+
+        verify {
+            navController.nav(
+                R.id.homeFragment,
+                HomeFragmentDirections.actionGlobalHomeSettingsFragment()
+            )
+        }
+    }
+
+    @Test
+    fun `GIVEN various sync account state WHEN Sync Account menu item is tapped THEN navigate to the appropriate sync fragment`() {
+        homeMenuBuilder.onItemTapped(HomeMenu.Item.SyncAccount(AccountState.AUTHENTICATED))
+
+        verify {
+            navController.nav(
+                R.id.homeFragment,
+                HomeFragmentDirections.actionGlobalAccountSettingsFragment()
+            )
+        }
+
+        homeMenuBuilder.onItemTapped(HomeMenu.Item.SyncAccount(AccountState.NEEDS_REAUTHENTICATION))
+
+        verify {
+            navController.nav(
+                R.id.homeFragment,
+                HomeFragmentDirections.actionGlobalAccountProblemFragment()
+            )
+        }
+
+        homeMenuBuilder.onItemTapped(HomeMenu.Item.SyncAccount(AccountState.NO_ACCOUNT))
+
+        verify {
+            navController.nav(
+                R.id.homeFragment,
+                HomeFragmentDirections.actionGlobalTurnOnSync()
+            )
+        }
+    }
+
+    @Test
+    fun `WHEN Bookmarks menu item is tapped THEN navigate to the bookmarks fragment`() {
+        homeMenuBuilder.onItemTapped(HomeMenu.Item.Bookmarks)
+
+        verify {
+            navController.nav(
+                R.id.homeFragment,
+                HomeFragmentDirections.actionGlobalBookmarkFragment(BookmarkRoot.Mobile.id)
+            )
+        }
+    }
+
+    @Test
+    fun `WHEN History menu item is tapped THEN navigate to the history fragment`() {
+        homeMenuBuilder.onItemTapped(HomeMenu.Item.History)
+
+        verify {
+            navController.nav(
+                R.id.homeFragment,
+                HomeFragmentDirections.actionGlobalHistoryFragment()
+            )
+        }
+    }
+
+    @Test
+    fun `WHEN Downloads menu item is tapped THEN navigate to the downloads fragment`() {
+        homeMenuBuilder.onItemTapped(HomeMenu.Item.Downloads)
+
+        verify {
+            navController.nav(
+                R.id.homeFragment,
+                HomeFragmentDirections.actionGlobalDownloadsFragment()
+            )
+        }
+    }
+
+    @Test
+    fun `WHEN Help menu item is tapped THEN open the browser to the SUMO help page`() {
+        homeMenuBuilder.onItemTapped(HomeMenu.Item.Help)
+
+        verify {
+            homeActivity.openToBrowserAndLoad(
+                searchTermOrURL = SupportUtils.getSumoURLForTopic(
+                    context = testContext,
+                    topic = SupportUtils.SumoTopic.HELP
+                ),
+                newTab = true,
+                from = BrowserDirection.FromHome
+            )
+        }
+    }
+
+    @Test
+    fun `WHEN Whats New menu item is tapped THEN open the browser to the SUMO whats new page and record metrics`() {
```

Comment:
nit: don't 100% remember, do we normally capitalize `and`s  after a `THEN` in test names?

Reply:
I think we do GIVEN WHEN THEN https://www.agilealliance.org/glossary/gwt/ (just a random reference I found from googling). It does not appear `AND` is part of it.